programs/pkey/pk_decrypt.c

```diff
@@ -118,11 +119,11 @@ int main( int argc, char *argv[] )
         goto exit;
     }
 
-    i = 0;
-
-    while( fscanf( f, "%02X", &c ) > 0 &&
-           i < (int) sizeof( buf ) )
-        buf[i++] = (unsigned char) c;
+    for( i=0; fscanf( f, "%02X", &c ) > 0 &&
```

Comment:
Style: please put space before and after the `=` operator.

Reply:
@andresag01 Done.